core/src/OCP/accessibility.js

```diff
@@ -29,4 +29,15 @@ export default {
 	disableKeyboardShortcuts() {
 		return loadState('theming', 'shortcutsDisabled', false)
 	},
+	/**
+	 * Set a page title to h1 header
+	 *
+	 * @param {string} [pageTitle] page title from the history api
+	 */
+	setPageTitle(pageTitle) {
```

Comment:
```suggestion
	setPageHeading(heading) {
```

because the title of a page is easily confused with document title